### PR TITLE
Layout polish and trying to figure out GWEI stuff

### PR DESCRIPTION
**Pull request by bconfortin**

### CODE REVIEW

File: packages/augur-ui/src/modules/modal/modal.styles.less [hunk 1]
```diff
@@ -1624,7 +1624,7 @@ div.ButtonsRow:last-of-type {
 .Gas {
   .Container;
 
-  max-width: 27.8125rem; // 445px
+  max-width: 30rem; // 480px
```
Comment (bconfortin) [inline packages/augur-ui/src/modules/modal/modal.styles.less:1627]: This was breaking in Safari, so I had to widen it up

File: packages/augur-ui/src/modules/modal/modal.styles.less [hunk 2]
```diff
@@ -1814,6 +1814,12 @@ div.ButtonsRow:last-of-type {
           border: @size-1 solid @color-secondary-action-outline;
           color: @color-primary-text;
           padding: @size-6 @size-16;
+          -moz-appearance:textfield;
+        }
+
+        input::-webkit-outer-spin-button,
+        input::-webkit-inner-spin-button {
+          -webkit-appearance: none;
```
Comment (bconfortin) [inline packages/augur-ui/src/modules/modal/modal.styles.less:1822]: This hides the arrows for the input[type='number']

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). ✨

 This link will always point to this PR's latest changes: https://peek.run/7g6hxxm

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**